src/app/accordion/accordion.sample.ts

```diff
@@ -0,0 +1,30 @@
+import { Component, ViewChild } from '@angular/core';
+import { slideInLeft, slideOutRight } from 'igniteui-angular';
+import { IgxAccordionComponent } from 'projects/igniteui-angular/src/lib/accordion/accordion.component';
```

Comment:
This should be changed as the accordion component is the public API - the import for `IgxAccordionComponent` should be from `'igniteui-angular'`